smart-contracts/contracts/merkle-incentives/.cargo/config

```diff
@@ -2,5 +2,5 @@
 wasm = "build --release --lib --target wasm32-unknown-unknown"
 unit-test = "test --lib"
 schema = "run --bin schema"
-test-tube = "test --package merkle-incentives --lib -- --include-ignored test_tube:: --nocapture --test-threads=1"
+test-tube = "test --package merkle-incentives --test merkle -- --include-ignored --nocapture --test-threads=1"
```

Comment:
Im not sure about the `--test merkle`, is that correct?

Reply:
oh nvm you moved the files to another folder

Reply:
Yeah, the imports work a bit different if we have all integration tests such as test-tube tests in the tests folder, but the testing modules also work just a bit nicer